storage/store.go

```diff
@@ -315,6 +315,9 @@ type StoreContext struct {
 
 	// EventFeed is a feed to which this store will publish events.
 	EventFeed *util.Feed
+
+	// Tracer is a request tracer used by this store for request tracing.
+	Tracer *util.Tracer
```

Comment:
Could we add the Tracer to the context.Context instead of threading another parameter throughout the codebase?